woocommerce/payment-gateway/apple-pay/class-sv-wc-payment-gateway-apple-pay-frontend.php

```diff
@@ -299,10 +340,15 @@ public function render_checkout_button() {
 
 		<div class="sv-wc-apply-pay-checkout">
 
-			<?php /** translators: Phrase that preceeds the Apple Pay logo, i.e. "Pay with [logo]" */
-			$button_text = __( 'Pay with', 'woocommerce-plugin-framework' );
+			<?php
+
+				/** translators: Phrase that preceeds the Apple Pay logo, i.e. "Pay with [logo]" */
+				$button_text = __( 'Pay with', 'woocommerce-plugin-framework' );
```

Comment:
you can also use the `_x` functions from WordPress to drop a disambiguation, e.g.

```
				$button_text = _x( 'Pay with', 'Phrase that preceeds the Apple Pay logo, i.e. "Pay with [logo]' 'woocommerce-plugin-framework' );
```

WordPress says (for these functions):

```
 * Quite a few times, there will be collisions with similar translatable text
 * found in more than two places, but with different translated context.
 *
 * By including the context in the pot file, translators can translate the two
 * strings differently.
 *
```

compare

https://developer.wordpress.org/apis/handbook/internationalization/internationalization-guidelines/#disambiguation-by-context

and

https://developer.wordpress.org/apis/handbook/internationalization/internationalization-guidelines/#descriptions

Seems `_x` & co. are preferable only when you have the same words in codebase but in different contexts or verb vs noun situations. A lengthier explanation could stay within `/* translators: */` note. In the specific case of this PR here then using the latter is correct.

Is this the right interpretation? Should we mention this in the wiki?

CC @ChaseWiseman

Reply:
I'm leaving this open so we can continue the discussion if needed, but this specific instance will not need the change as the variable wasn't being used and was removed.

Reply:
@unfulvio @deivamagalhaes yeah we've already got a good set of guidelines for context, and using `_x()` vs. `__()` + translator comment here: https://github.com/skyverge/wc-plugins/wiki/WordPress-and-WooCommerce-Best-Practices#context

Based on that, `__()` w/ the translator comment seems correct, if we were keeping that line.